modules/EvseV2G/connection.cpp

```diff
@@ -985,3 +1022,51 @@ int connection_start_servers(struct v2g_context* ctx) {
 
     return 0;
 }
+int create_udp_socket(const u_int16_t udp_port) {
```

Comment:
:+1: , still return values of `inet_pton` and `connect` should be checked

Reply:
Why should this be a multicast address? Also as far as I'm aware if we use this address I need to specify a network interface. Is this what we need?

Reply:
We should use the same interface for the UDP message as for the ISO communication. Otherwise it can happen that a different interface is selected by the OS for sending this debug udp message. And when debugging the ISO messages via a pcap file, we will not be able to decrypt the TLS messages.

Reply:
Exactly, it must be on the same interface - also to not accidentally propagate. The secret is only supposed to be visible on this host, possibly the communication partner, and of course the captured trace.

I think a solution might be to also use `bind()`. When using it for the UDP source port, it can also bind to the source address, to the address of the TLS connection.